tools/webtransport/h3/webtransport_h3_server.py

```diff
@@ -35,11 +36,55 @@
 _doc_root: str = ""
 
 
+class CapsuleType(IntEnum):
+    # Defined in
+    # https://www.ietf.org/archive/id/draft-ietf-masque-h3-datagram-03.html.
+    DATAGRAM = 0xff37a0
+    REGISTER_DATAGRAM_CONTEXT = 0xff37a1
+    REGISTER_DATAGRAM_NO_CONTEXT = 0xff37a2
+    CLOSE_DATAGRAM_CONTEXT = 0xff37a3
+    # Defined in
+    # https://www.ietf.org/archive/id/draft-ietf-webtrans-http3-01.html.
+    CLOSE_WEBTRANSPORT_SESSION = 0x2843
+
+
+class H3Capsule:
+    """
+    Represents the Capsule concept defined in
+    https://ietf-wg-masque.github.io/draft-ietf-masque-h3-datagram/draft-ietf-masque-h3-datagram.html#name-capsules.
+    """
+    def __init__(self, type: int, data: bytes) -> None:
+        self.type = type
+        self.data = data
+
+    @staticmethod
+    def decode(data: bytes) -> any:
```

Comment:
I'm using python3.8 but it complains at H3Capsule here.

Reply:
I see. I thought [PEP 563](https://www.python.org/dev/peps/pep-0563/) should work for Python 3.8 but I forgot that we need to add `from __future__ import annotations` to enable it. I dropped it since it didn't work for Python 3.6. Thank you for clarification.